src/components/app/app.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { fireEvent, render, screen, within } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import { App } from './app'
+import React from 'react'
+import * as deck from '../../services/deck'
+import { CardSuit, CardValue } from '../../interfaces/card.interface'
+
+test('Renders app', () => {
+    render(<App />)
+
+    expect(screen.getByText('DEALER')).toBeInTheDocument()
+    expect(screen.getByText('PLAYER')).toBeInTheDocument()
+})
+
+test('Dealer is dealt a seven and ten', () => {
+    mockDealHand(CardValue.SEVEN, CardValue.TEN)
+    mockDealHand(CardValue.SEVEN, CardValue.TEN)
+
+    render(<App />)
+
+    fireEvent.click(screen.getByText('STAND'))
+
+    expect(
+        screen.getByText('DEALER').nextSibling.firstChild.nextSibling
+            .nextSibling.textContent
+    ).toBe('Total:  17')
+})
+
+test('Player is dealt two kings', () => {
+    mockDealHand(CardValue.SEVEN, CardValue.KING)
+    mockDealHand(CardValue.KING, CardValue.KING)
+
+    render(<App />)
+
+    fireEvent.click(screen.getByText('SPLIT'))
+
+    expect(screen.getByText('PLAYER').nextSibling.childNodes.length).toBe(2)
```

Comment:
don't really have easy way to avoid depending on nextSibling.childNodes for now so we can leave as is